components/post_view/message_attachments/message_attachment/message_attachment.jsx

```diff
@@ -365,6 +366,46 @@ export default class MessageAttachment extends React.PureComponent {
             );
         }
 
+        let footer;
+        if (attachment.footer) {
+            let footerIcon;
+            if (attachment.footer_icon) {
+                const footerIconMetadata = this.props.imagesMetadata[attachment.footer_icon];
+
+                footerIcon = (
+                    <ExternalImage
+                        src={attachment.footer_icon}
+                        imageMetadata={footerIconMetadata}
+                    >
+                        {(footerIconUrl) => (
+                            <img
+                                alt={'attachment footer icon'}
+                                className='attachment__footer-icon'
+                                src={footerIconUrl}
+                                height='16'
+                                width='16'
+                            />
+                        )}
+                    </ExternalImage>
+                );
+            }
+
+            let truncatedFooter = attachment.footer;
+            if (truncatedFooter.length > Constants.MAX_ATTACHMENT_FOOTER_LENGTH) {
+                truncatedFooter = attachment.footer.substring(0, Constants.MAX_ATTACHMENT_FOOTER_LENGTH) +
+                    localizeMessage('message_attachment.ellipsis', '…');
```

Comment:
@hanzei I wasn't sure if the ellipsis (…) needed to be `i18n`'d - is this the right way to do this?

Reply:
Good question. I don't know. @jasonblais Can you help with this question or know who can answer?

Reply:
Good question indeed. From my perspective they won't need to be localized but @jespino wondering how your translations tool (via `make i18n-extract`) handles ellipsis? Would be good to stay consistent with that.

Reply:
Is not handled in any special way, it is just text like any other. I'm not sure if we want to handled it, but for sure I wouldn't localized it as an "special" entity, the main problem with localization is you need to understand the context to be able to translate a thing, for example you can't "localize words" and concatenate them, because the order and the context of the words can lead to very different translations.

In this case we are talking about truncating a "dynamic" string without any concrete language, and in general the `...` is pretty universal, and at least is visually self-explanatory.

I would prefer to not localize it at all here, and actually we can use directly the `lodash` `truncate` function (https://lodash.com/docs/4.17.11#truncate).

Reply:
Thanks @jespino!  I've removed the i18n on the ellipsis and swapped out the `String.substring` truncation for `lodash.truncate`.